.github/workflows/test.yml

```diff
@@ -26,6 +26,7 @@ jobs:
             vhpidirect/shared/shlib,
             vhpidirect/shared/dlopen,
             vhpidirect/shared/shghdl,
+            vhpidirect/cinterface/accarraygen,
```

Comment:
I think this belongs to `arrays/intvector` (as a subexample) or `arrays/intvectorgen`.

Reply:
I'll probably group it under intvector.